Expand Difficulty Reduction mod articles + Add "Mods (lazer)" on the Main page (#12600)
ppy/osu-wiki: wiki/Gameplay/Game_modifier/Half_Time_(lazer)/en.md

*For the osu!stable version of this article, see: [Half Time (mod)](/wiki/Gameplay/Game_modifier/Half_Time)*\
*For the full list of all [lazer](/wiki/Client/Release_stream/Lazer) mods, see: [Game modifier (lazer)](/wiki/Gameplay/Game_modifier_(lazer))*

> The **Half Time** mod decreases the BPM of any beatmap by 75% as well as increasing the length of the song by 33%. It also can reduce the [approach rate (AR)](/wiki/Beatmap/Approach_rate), [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) or both at once according to the [game mode](/wiki/Game_mode).

Reviewer: "decrease by 75%" means "multiply by 0.25"

```suggestion
The **Half Time** mod decreases the BPM of any beatmap to 75%, increasing the length of the song by 33%. It also can reduce the [approach rate (AR)](/wiki/Beatmap/Approach_rate), [overall difficulty (OD)](/wiki/Beatmap/Overall_difficulty) or both at once depending on the [game mode](/wiki/Game_mode).
```

## Customisation

![Half Time customisation options in the game client](/wiki/Gameplay/Game_modifier_(lazer)/img/customise/HT.jpg)

> - **Speed decrease** (0,50x-0,99x, default 0,75x): The speed at which the map is played.

Reviewer: ```suggestion
- **Speed decrease** (0.50x–0.99x, default 0.75x): The speed at which the map is played.
```

> - **Adjust pitch** (default disabled): Changes the audio frequency depending on the chosen speed. *Enabling this option doesn't make the mod unrankable.* (playing with the default speed will have the same audio effect as [Daycore](/wiki/Gameplay/Game_modifier/Daycore))

Reviewer: ```suggestion
- **Adjust pitch** (disabled by default): Changes the audio frequency depending on the chosen speed. Enabling this option does not make the play unranked. (Enabling pitch adjustment with the default speed will have the same effect as [Daycore](/wiki/Gameplay/Game_modifier/Daycore).)
```
